Harvest.Web/ClientApp/src/App.tsx

```diff
@@ -30,30 +30,33 @@ function App() {
     <AppContext.Provider value={Harvest}>
       <Route exact path="/" component={Home} />
       <Route exact path="/home/spa" component={Spa} />
-      <Route path="/request/create/:projectId?" component={RequestContainer} />
-      <Route path="/request/approve/:projectId" component={ApprovalContainer} />
-      <Route
+      <ConditionalRoute roles={["FieldManager"]} path="/request/create/:projectId?" component={RequestContainer} />
+      <ConditionalRoute roles={["FieldManager"]} path="/request/approve/:projectId" component={ApprovalContainer} />
```

Comment:
The PI is probably the only one who should be approving requests.  Maybe a FM should be able to in emergency scenarios, but the PI is the main one.

Reply:
Request or quote? FM should approve requests, PI the quote.

Reply:
@jSylvestre I don't understand, what is the FM approving?  The only approval button is to approve the quote, which is /request/approve/ url

Reply:
Ah. Just me being confused. When you said approving requests, I was thinking project not quote

Reply:
Fixed